src_new/config.js

```diff
@@ -261,18 +279,41 @@ exports.updatePWTConfig = function () {
 				config[CONSTANTS.CONFIG.COMMON][key] = testConfig[key];
 			}
 		}
+		// Uncomment Below code after updating phatomjs or using chrome headless 
+		// Object.assign(config[CONSTANTS.CONFIG.COMMON], testConfig);
 		window.PWT.testGroupId = 1;
 	}
 };
 
-exports.isAbTestEnabled = function () {
-	return parseInt(config[CONSTANTS.CONFIG.COMMON][CONSTANTS.CONFIG.AB_TEST_ENABLED]) === 1;
-};
-
-exports.getTestPWTConfig = function () {
-	return config[CONSTANTS.COMMON.TEST_PWT] || {};
+exports.updatePartnerConfig = function (testConfig, controlConfig) {
+	if (testConfig && Object.keys(testConfig).length > 0) {
+		util.log(CONSTANTS.MESSAGES.M31, JSON.stringify(testConfig));
+		for (var key in testConfig) {
+			if (util.isOwnProperty(testConfig, key) && util.isObject(testConfig[key])) {
+				if (Object.keys(testConfig[key]).length == 0 && controlConfig[key] && Object.keys(controlConfig[key]).length > 0) {
+					testConfig[key] = controlConfig[key];
+				} else if (Object.keys(testConfig[key]).length > 0 && controlConfig[key] && Object.keys(controlConfig[key]).length > 0) {
+					testConfig[key] = refThis.getMergedConfig(testConfig[key], controlConfig[key]);
+				}
+			}
+		}
+		window.PWT.testGroupId = 1;
+		return testConfig;
+	} else{
+		return {};
```

Comment:
Since there will be a single AB Test type only, we need to return control partners if there  are no test type

Reply:
Done